app/controllers/rss_logs_controller.rb

```diff
@@ -73,7 +73,7 @@ def show_selected_rss_logs(query, args = {})
       glossary_term: :user,
       location: :user,
       name: :user,
-      observation: [:location, :name, :user,
+      observation: [:location, :name, :namings, :user,
                     @user ? { thumb_image: :image_votes } : :thumb_image],
```

Comment:
Note the new `includes(:namings)` requested by Bullet here.

Reply:
Ah, ok. This extra `includes(:namings)` significantly slows down the page load; `namings` table lookups are an order of magnitude slower than everything else.
 
I'd like to figure out where i'm loading namings for the lightbox caption info, and hopefully get rid of it, before merging this.

Reply:
Resolved by moving a `Textile.register_name` line in the partial. (Note below)